checks/evaluation/signed_releases.go

```diff
@@ -47,10 +47,13 @@ func SignedReleases(name string,
 		f := &findings[i]
 
 		// Debug release name
```

Comment:
We just need to move this block higher up:
```go
		// Check if outcome is NotApplicable
		if f.Outcome == finding.OutcomeNotApplicable {
			dl.Warn(&checker.LogMessage{
				Text: "no GitHub releases found",
			})
			// Generic summary.
			return checker.CreateInconclusiveResult(name, "no releases found")
		}
```

Reply:
note: this `Text` says `no GitHub releases found`, but really should just be `no releases found`